droidlet/dashboard/web/config/webpack.config.js

```diff
@@ -53,6 +53,10 @@ const sassModuleRegex = /\.module\.(scss|sass)$/;
 // This is the production and development configuration.
 // It is focused on developer experience, fast rebuilds, and a minimal bundle.
 module.exports = function(webpackEnv) {
+  watchOptions: {
```

Comment:
HiTL dashboard is different from Droidlet dashboard. This PR shouldn't touch any files under `droidlet/dashboard`, right?

Reply:
Yeah, thanks for pointing this out, I was testing on the previous dashboard before, but forgot to change it back. Let me revert this change.